rx_signal_processing/dsp.cu

```diff
@@ -828,44 +875,91 @@ float DSPCore::get_decimate_timing()
   return decimate_kernel_timing_ms;
 }
 
+/**
+ * @brief      Gets the host pointer for first stage output.
+ *
+ * @return     The first stage output host pointer.
+ */
 cuComplex* DSPCore::get_first_stage_output_h()
 {
   return first_stage_output_h;
 }
 
+/**
+ * @brief      Gets the host pointer for the second stage output.
+ *
+ * @return     The second stage output host pointer.
+ */
 cuComplex* DSPCore::get_second_stage_output_h()
 {
   return second_stage_output_h;
 }
 
+/**
+ * @brief      Gets the host pointer for the third stage output.
+ *
+ * @return     The third stage output host pointer.
+ */
 cuComplex* DSPCore::get_third_stage_output_h()
 {
   return third_stage_output_h;
 }
 
+/**
+ * @brief      Gets the number of antennas.
+ *
+ * @return     The number of antennas.
+ */
 uint32_t DSPCore::get_num_antennas()
 {
   return num_antennas;
 }
 
-uint32_t DSPCore::get_num_rf_samples() {
+/**
+ * @brief      Gets the number of rf samples.
+ *
+ * @return     The number of rf samples.
+ */
+uint32_t DSPCore::get_num_rf_samples()
+{
   return num_rf_samples;
 }
+
+/**
+ * @brief      Gets the number first stage samples per antenna.
+ *
+ * @return     The number first stage samples per antenna.
+ */
 uint32_t DSPCore::get_num_first_stage_samples_per_antenna()
 {
   return num_first_stage_samples_per_antenna;
 }
 
+/**
+ * @brief      Gets the number second stage samples per antenna.
+ *
+ * @return     The number second stage samples per antenna.
+ */
 uint32_t DSPCore::get_num_second_stage_samples_per_antenna()
 {
   return num_second_stage_samples_per_antenna;
 }
 
+/**
+ * @brief      Gets the number third stage samples per antenna.
+ *
+ * @return     The number third stage samples per antenna.
+ */
 uint32_t DSPCore::get_num_third_stage_samples_per_antenna()
 {
   return num_third_stage_samples_per_antenna;
 }
 
+/**
+ * @brief      Gets the sequence number.
```

Comment:
Maybe explain briefly what the sequence number is for "Used as identifier/error check between code blocks'